### PR TITLE
Feature/te 424 aml syntax should be more compact

### PR DESCRIPTION
**Pull request by kpanier**
- Shorter syntax for aml
- More usage of short syntax in uatests
- More usage of locator strategy in uatests
- minor fixture imrpovements

### CODE REVIEW

File: rcp/org.testeditor.rcp4.uatests/src/main/java/org/testeditor/fixture/swt/SWTFixture.java [hunk 1]
```diff
@@ -154,9 +161,12 @@ public void waitForDialogClosing(String title) {
...
 		logger.info("Waiting for dialog with title='{}' to open, timeout='{}' seconds.", title, timeout);
 		try {
-			SWTBotShell shell = bot.shell(title);
-			if (shell.isActive()) {
+			try {
+				bot.waitUntil(Conditions.shellIsActive(title), timeout * 1000);
+				SWTBotShell shell = bot.shell(title);
 				bot.waitUntil(Conditions.shellCloses(shell), timeout * 1000);
+			} catch (TimeoutException e) {
+				logger.info("Dialog with title {} was not found. Test continuous.", title);
```
Comment (gunther-bachmann) [inline rcp/org.testeditor.rcp4.uatests/src/main/java/org/testeditor/fixture/swt/SWTFixture.java]: Shouldn't this be at least a warning?

File: tcl/org.testeditor.tcl.sdk/feature.xml [hunk 2]
```diff
@@ -59,4 +59,18 @@ itemis AG
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
+         id="org.testeditor.dsl.common.model"
```
Comment (franzbecker) [inline tcl/org.testeditor.tcl.sdk/feature.xml]: Shouldn't we have a separate feature for the common parts?

File: aml/org.testeditor.aml.dsl/src/org/testeditor/aml/dsl/Aml.xtext [hunk 3]
```diff
@@ -53,12 +53,12 @@ ComponentType returns ComponentType:
...
 
 ComponentElement returns ComponentElement:
-	'element' name=ID 'is' type=[ComponentElementType|QualifiedName] ('{'
+	'element' name=ID 'is' type=[ComponentElementType|QualifiedName] (('{'
 		('label' '=' label=STRING)?
 		('locator' '=' locator=STRING)?
 		('locatorStrategy' '=' locatorStrategy=[types::JvmEnumerationLiteral|QualifiedName])?
 		(valueSpaceAssignments+=ValueSpaceAssignment)*
-	'}')?
+	'}')|('locate' 'by' locatorStrategy=[types::JvmEnumerationLiteral|QualifiedName] ('using')? locator=STRING)(valueSpaceAssignments+=ValueSpaceAssignment)*)?
```
Comment (franzbecker) [inline aml/org.testeditor.aml.dsl/src/org/testeditor/aml/dsl/Aml.xtext]: Could you improve the formatting of this rule?

Comment (franzbecker) [inline aml/org.testeditor.aml.dsl/src/org/testeditor/aml/dsl/Aml.xtext]: I'm not sure about the `valueSpaceAssignments`. Do we really want them in this short syntax?
If yes, a parser test is missing for them.

File: tcl/org.testeditor.tcl.sdk/feature.xml (hunk 2 above)
Comment (kpanier) [inline tcl/org.testeditor.tcl.sdk/feature.xml]: +1 but in a new story

**Pull request merged**